docs/distributions/gentoo/installation.md

```diff
@@ -1,89 +1,46 @@
+   emerge -av sys-kernel/t2gentoo-kernel
+   ```
 
-    ```bash
-    git clone https://github.com/Redecorating/mbp-16.1-linux-wifi /linux-patches
-    cd /usr/src/linux
-    for i in /linux-patches/*.patch; do
-    echo $i
-    patch -Np1 < $i
-    done
-    ```
+7. Install WiFi firmware for T2 macs:
 
-    1. Git clone the `https://anongit.gentoo.org/git/proj/linux-patches.git` repo to a folder called `/gentoo-patches`. This repo includes Gentoo patches for the kernel source tree usually included with the `gentoo-sources` Portage package.
-    2. Apply them with a modified version of the commands used above.
-    3. Copy the default config from the patches repo to the kernel source tree. If you do this, please make sure to set any filesystem drivers you want to use (like for ext4) to be built-in instead of being a module with `make menuconfig`
-    4. Build with `make all`. If you want to speed up the build process, add `-j$(nproc)`
+   ```bash
+   mkdir -p /etc/portage/package.license
+   echo "sys-firmware/apple-bcm-firmware all-rights-reserved" >> /etc/portage/package.license/firmware
+   emerge -av sys-firmware/apple-bcm-firmware
+   ```
 
-7. Before finishing, if you want to connect to the internet later, now is a good time to install `NetworkManager` and optionally `iwd`.
-8. Continue through the Handbook until chapter "Configuring the bootloader"
+8. Follow the Gentoo Handbook from [Configuring the System](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/System) up to [Configuring the bootloader](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Bootloader)
 
-    1. Install grub by using `emerge --ask --verbose sys-boot/grub`
-    2. Edit the config file `/etc/default/grub` with your favorite text editor of choice (i.e. vim or nano)
-    3. On the line with `GRUB_CMDLINE_LINUX`, append the following parameters: `intel_iommu=on iommu=pt pcie_ports=compat`
-    4. Run `grub-install --target=x86_64-efi --efi-directory=/boot/efi --bootloader-id=GRUB --no-nvram --removable` to install Grub.
-    5. Run `grub-mkconfig -o /boot/grub/grub.cfg` to make the config files for Grub
+9. Install the bootloader:
+    1. Choose a bootloader (other than LILO) from the [Configuring the bootloader](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Bootloader) section of the Gentoo Handbook.
+    2. Add `intel_iommu=on iommu=pt pcie_ports=compat` to the kernel parameters. Refer to the Gentoo Wiki article for your bootloader of choice for instructions on how to do this.
 
-9. You're done! You should now be able to boot into Gentoo via the macOS Startup Manager
-10. If you confirmed that Gentoo does bootup no problem, then you can boot into macOS and follow the [Wi-Fi guide](https://wiki.t2linux.org/guides/wifi/)
+10. Exit the `chroot` (Control-d, or `exit`) and reboot. You should now be able to select Gentoo from the MacOS startup manager by holding option at boot. Congratulations, you should now have a working Gentoo installation.
```

Comment:
macOS, not MacOS. Also, you probably could add a ! Instead of a full stop at the end of this line.